ts/features/wallet/components/WalletCardsContainer.tsx

```diff
@@ -64,17 +69,22 @@ const WalletCardsContainer = () => {
     }
     return (
       <View testID="walletCardsContainerTestID" style={IOStyles.flex}>
-        <ItwWalletCardsContainer />
+        {!isWalletInstanceStatusFailure && <ItwWalletCardsContainer />}
```

Comment:
Can we handle the render logic inside `ItwWalletCardsContainer`? Just for consistency, we are doing the same with `isItwEnabled`